ui-v2/GNUmakefile

```diff
@@ -7,6 +7,9 @@ dist:
 	yarn run build
 	mv dist ../pkg/web_ui/v2
```

Comment:
Ah keep meaning to ask, would it be ok to move this `mv` up to whatever calls this instead of in here, no biggie if not. Basically if we move this out it means this Makefile/folder is pretty much self contained. Ping me later and we can chat.

Reply:
definitely, I think I had that change until I pulled master for the rsync change and since we both modified it I just resolved with master and redid the bits to add a `make init`